File: core/src/main/java/io/tcc/core/service/dto/PostListDTO.java
package io.tcc.core.service.dto;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;
import java.util.List;
import java.util.UUID;

@Getter
@Setter
public class PostListDTO {

    private UUID id;
    private LocalDateTime postDate;
    private String title;
    private String description;
    private Long score;
    private PostUserDTO user;
    private String subject;
    private Integer status;
    private List<String> tags;

}

File: front/src/model/PostListDTO.ts
import {DocumentSaveDTO} from "./DocumentSaveDTO";
import {PostUserDTO} from "./PostUserDTO";

export class PostListDTO {

  id?: string;

  postDate?: string;

  title?: string;

  description?: string;

  score?: string;

  user?: PostUserDTO;

  subject?: number;

  status?: number;

  tags?:string[];

  images?: DocumentSaveDTO[];

}
